prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -264,8 +267,8 @@ static int fi_ibv_rdm_av_remove(struct fid_av *av_fid, fi_addr_t * fi_addr,
 		return -FI_EINVAL;
 	}
 
-	if (av->ep) {
-		pthread_mutex_lock(&av->ep->cm_lock);
+	if (ep) {
```

Comment:
I just see that single if statement are wrapped by the braces in verbs prov
Done